next/components/forms/segments/AccountSectionHeader/TaxFeeSectionHeader.tsx

```diff
@@ -0,0 +1,140 @@
+import FileDownload from '@assets/images/account/file_download.svg'
+import PaymentIcon from '@assets/images/account/payment-icon.svg'
+import ChevronLeft from '@assets/images/chevron-left-2.svg'
+import ExclamationIcon from '@assets/images/forms/exclamation-icon.svg'
+import SuccessIcon from '@assets/images/forms/success.svg'
+import TimeIcon from '@assets/images/forms/warning-time-icon.svg'
+import cx from 'classnames'
+import { useRouter } from 'next/router'
+import { useTranslation } from 'next-i18next'
+import { ReactNode } from 'react'
+
+import Button from '../../simple-components/Button'
+
+type AccountSectionHeaderBase = {
+  title: string
+  status?: string
+}
+
+const statusHandler = (status: 'negative' | 'warning' | 'success'): ReactNode => {
+  const statusStyle: string = cx('lg:text-p2-semibold text-p3-semibold', {
+    'text-negative-700': status === 'negative',
+    'text-warning-700': status === 'warning',
+    'text-success-700': status === 'success',
+  })
+  const statusNode = (icon: ReactNode, statusTitle: string): ReactNode => {
+    return (
+      <>
+        <span className="h-6 w-6 flex justify-center items-center">{icon}</span>
+        <span className={statusStyle}>{statusTitle}</span>
+      </>
+    )
+  }
+
+  switch (status) {
+    case 'negative':
+      return statusNode(<ExclamationIcon />, 'Neuhradená')
+    case 'warning':
+      return statusNode(<TimeIcon />, 'Čiastočne uhradená')
+    case 'success':
+      return statusNode(<SuccessIcon width={18} height={24} viewBox="0 0 24 18" />, 'Uhradená')
+
+    default:
+      return null
+  }
+}
+
+const priceFormat = (price: number): string => {
+  return Number.isInteger(price) ? `${price},00 €` : `${price} €`.replace('.', ',')
+}
```

Comment:
would rather not have assumptions about the format this comes from the server - please remove this fn and in place where you use it, just keep a placeholder text